components/feature/push/src/main/java/mozilla/components/feature/push/AutoPushFeature.kt

```diff
@@ -101,27 +101,21 @@ class AutoPushFeature(
 
     private val coroutineScope = CoroutineScope(coroutineContext) + SupervisorJob() + exceptionHandler { onError(it) }
 
```

Comment:
I moved this initialization code to the explicit `initialize` method, mainly for testing